### PR TITLE
Fix error reporting on diff.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

**Comment by stackenbotten** [comment]
There was an error when running `chk_coding_style` for commit `28f459cfab0b2be3d1d812bb622ed5e47a876d7b`:
```
Coding style error:
 libsolidity/codegen/ExpressionCompiler.cpp:1588: for (auto& function : contract.definedFunctions()) {
 libsolidity/codegen/ExpressionCompiler.cpp:1589: const u256 hash(util::keccak256(function->externalSignature()));

```
Please check that your changes are working as intended.

File: libsolidity/codegen/ExpressionCompiler.cpp [hunk 1]
```diff
@@ -1580,6 +1580,20 @@ bool ExpressionCompiler::visit(MemberAccess const& _memberAccess)
 			m_context << Instruction::DUP1 << u256(32) << Instruction::ADD;
 			utils().storeStringData(contract.name());
 		}
+		else if (member == "interfaceId")
+		{
+			TypePointer arg = dynamic_cast<MagicType const&>(*_memberAccess.expression().annotation().type).typeArgument();
+			ContractDefinition const& contract = dynamic_cast<ContractType const&>(*arg).contractDefinition();
+			u256 result{0};
+			for (auto& function : contract.definedFunctions()) {
```
Comment (stackenbotten) [inline libsolidity/codegen/ExpressionCompiler.cpp]: Coding style error

**Comment by leonardoalt** [comment]
What was the problem before?

**Comment by chriseth** [comment]
The problem was that it reported it on the "left" (original) part of the diff, but it is actually computed on the "right" (changed) part of the diff.

**Review by leonardoalt — APPROVED** [review]

**Pull request merged**